airbyte-webapp/src/components/GeographyDropdown/GeographyDropdown.tsx

```diff
@@ -0,0 +1,64 @@
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { EU, US } from "country-flag-icons/react/3x2";
+import React from "react";
+import { components, OptionProps, MenuListProps } from "react-select";
+
+import { DropDown } from "components/ui/DropDown";
+
+import styles from "./GeographyDropdown.module.scss";
+
+const options = [
+  {
+    value: "us",
+    label: "us",
+  },
+  {
+    value: "auto",
+    label: "auto",
+  },
+  {
+    value: "eu",
+    label: "eu",
+  },
+];
+
+const flags: Record<string, React.ReactNode> = {
+  auto: <US />,
+  us: <US />,
+  eu: <EU />,
```

Comment:
This way will require us to extend with new icons every time the API would return new region codes. I wonder if it might actually be better to use a library that does not require us to "import" the components, but assigning just some CSS classname, and we could just insert whatever ISO code we got from the backend there, e.g. https://www.npmjs.com/package/flag-icons

That way we would not need this map at all and would just have a `<span className={`fi fi-${label === "auto" ? "us" : label}`} />`.

@josephkmh your thoughts?

Reply:
I think that's a good approach! Easier than maintaining a map ourselves.

Reply:
That is an excellent point @timroes ! @josephkmh and I talked about this issue, this library looks like a better solution than the one I am currently using.